packages/devtools_app/pubspec.yaml

```diff
@@ -52,7 +52,7 @@ dependencies:
   stack_trace: ^1.12.0
   string_scanner: ^1.1.0
   unified_analytics: ^7.0.0
-  vm_service: '>=13.0.0 <16.0.0'
+  vm_service: ">=15.0.0 <16.0.0"
```

Comment:
does 15.0.0 come with a min sdk version bump?

Reply:
No, since it wasn't a change in the service protocol. There's a breaking change in `package:vm_service` where `Code.function` is no longer `FuncRef` but `dynamic` since it can also be a `NativeFunction`. This impacts code in DevTools, which is why I'm raising the lower bound to the newest compatible version.